Content.Client/Power/Visualizers/CableVisualizerSystem.cs

```diff
@@ -1,20 +1,11 @@
-using Content.Client.SubFloor;
-using Content.Shared.SubFloor;
 using Content.Shared.Wires;
 using Robust.Client.GameObjects;
 
 namespace Content.Client.Power.Visualizers;
 
-public sealed partial class CableVisualizerSystem : EntitySystem
+public sealed class CableVisualizerSystem : VisualizerSystem<CableVisualizerComponent>
 {
-    public override void Initialize()
-    {
-        base.Initialize();
-
-        SubscribeLocalEvent<CableVisualizerComponent, AppearanceChangeEvent>(OnAppearanceChanged, after: new[] { typeof(SubFloorHideSystem) });
-    }
-
-    private void OnAppearanceChanged(EntityUid uid, CableVisualizerComponent component, ref AppearanceChangeEvent args)
```

Comment:
Pretty sure removing this cause bugs with wire visualization

Reply:
I missed the `after:`, that is probably there for a reason yeah.